frontend/src/components/homepage/StoryCard.tsx

```diff
@@ -155,7 +159,7 @@ const StoryCard = ({
             )}`}</Badge>
           </Flex>
         </Flex>
-        <Text size="xs">{description}</Text>
+        <Text size="xs">{truncateDescription()}</Text>
```

Comment:
maybe pass description as parameter i.e truncateText(string description) and optionally length as well